source/updateCheck.py

```diff
@@ -500,7 +504,6 @@ def _error(self):
 			wx.OK | wx.ICON_ERROR)
 
 	def _downloadSuccess(self):
-#		global storeUpdatesDir

```

Comment:
Should this be uncommented / removed?

Reply:
This has already been removed. Probably legacy code from the initial author, but storeUpdatesDir doesn't have to be changed inside this function.

Reply:
Oh sorry I must have misread this line.